internal/commands/reminder.go

```diff
@@ -28,30 +28,28 @@ func requestStatus(ctx context.Context, app *app.App, jobIncident model.Incident
 				ctx,
 				log.Trace(),
 				log.Reason("GetIncident"),
-				log.NewValue("channelID", incident.ChannelId),
-				log.NewValue("channelName", incident.ChannelName),
 				log.NewValue("error", err),
 			)
 			return
 		}
 
-		app.Logger.Info(
+		logWriter := app.Logger.With(
```

Comment:
Shouldn't `logWriter` declaration be before the `GetIncident` error? Notice that you removed the fields `ChannelID` & `ChannelName` from this error, but it is still using `app.Logger` instead of `logWriter`.

Reply:
Oh, no, I just saw that the removed fields come from the incident object, returned by `GetIncident`, so what you did is correct.